python/ray/ray_perf.py

```diff
@@ -22,6 +23,22 @@ def small_value_batch(self, n):
         ray.get([small_value.remote() for _ in range(n)])
 
 
+@ray.remote
+class AsyncActor:
+    async def private_coroutine(self):
```

Comment:
I don't think we even need this, it's adding some noise to the benchmark. What we want this to show is how much overhead async actors have vs. sync actors, and this is adding some additional overhead from asyncio coroutines. I would just have the handlers return immediately like the other benchmarks.

Reply:
I actually tested both versions with / without it, and the perf looks almost identical (sync one was ~1% better). But I understand this will be noise. Will remove it!

Reply:
Fixed!